github-actions/verify-pr-title/package.json

```diff
@@ -17,14 +17,13 @@
     "scripts": {
         "bundle": "npm run format:write && npm run package",
         "ci-test": "npx jest",
-        "coverage": "npx make-coverage-badge --output-path ./badges/coverage.svg",
         "format:write": "npx prettier --write .",
         "format:check": "npx prettier --check .",
         "lint": "npx eslint src/**/*.ts",
         "package": "npx ncc build src/index.ts -o dist --source-map --license licenses.txt",
         "package:watch": "npm run package -- --watch",
         "test": "npx jest",
-        "all": "npm run format:write && npm run lint && npm run test && npm run coverage && npm run package"
```

Comment:
The coverage command no longer makes any sense and was causing a compile failure, so I removed it.